src/main/java/com/github/checkstyle/regression/extract/CheckstyleInjector.java

```diff
@@ -139,14 +139,13 @@ private void invokeMavenCommand() throws InjectException {
      * @throws IOException failure of copying
      */
     private void copyInjectFilesToCheckstyleRepo() throws IOException {
-        final File srcDir =
-                new File("src/main/resources/com/github/checkstyle/regression/extract/");
+        final String path = "/com/github/checkstyle/regression/extract/";
         final File destDir = new File(repoPath, "src/test/java/com/puppycrawl/tools/checkstyle/");
         final String[] injections = {"ExtractInfoGeneratorTest.java", "JsonUtil.java"};
```

Comment:
We probably should have a test to verify all files in extract directory are listed in this string.
Please make a new issue.